docs/swagger.yml

```diff
@@ -0,0 +1,238 @@
+# Available at the /docs endpoint.
+paths:
+  /achievements:
+    get:
+      parameters:
+        - description: Number of results to return
+          in: query
+          name: $limit
+          type: integer
+        - description: Number of results to skip
+          in: query
+          name: $skip
+          type: integer
+        - description: Property to sort results
+          in: query
+          name: $sort
+          type: string
+      responses:
+        '200':
+          description: success
+          schema:
+            $ref: '#/definitions/achievements_list'
+        '401':
+          description: not authenticated
+        '500':
+          description: general error
+      description: Retrieves a list of all resources from the service.
```

Comment:
Resources? What resources? This seems like a general description for get and not specific to each request?